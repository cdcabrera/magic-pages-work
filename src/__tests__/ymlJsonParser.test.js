const { ymlJsonParser } = require('../ymlJsonParser');

describe('ymlJsonParser', () => {
  it('should attempt to parse a Swagger spec file', async () => {
    // let output;

    const output = await ymlJsonParser({
      files: [
        {
          source: 'petStoreSwagger.yml'
        }
      ],
      parseMethod: 'bundle'
    });

    console.log('>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>');
    console.log('output', Array.isArray(output));
    console.log('>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>');

    expect(output).toMatchSnapshot('swagger');
  });
});
