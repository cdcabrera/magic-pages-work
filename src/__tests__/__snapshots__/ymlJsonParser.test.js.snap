// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ymlJsonParser should attempt to parse a Swagger spec file: swagger 1`] = `
[
  {
    "content": undefined,
    "output": undefined,
    "source": "../src/__fixtures__/petStoreSwagger.yml",
  },
]
`;
